refarch-frontend/.eslintrc.js

```diff
@@ -20,7 +20,13 @@ module.exports = {
     "@vue/eslint-config-prettier",
   ],
   rules: {
-    "no-console": "error",
+    "no-console": ["error", { allow: ["debug"] }],
```

Comment:
why do we need to allow that for debug? And does debug even exist, i only found the values `log|warn|error|info|trace`

Reply:
Browsers nowadays have a specific debug mode you can activate. I think its a good practice to allow console.debug statements even in production code. I got this from a different project. This way the console in normal use stays clear. But you can still debug behaviour and get additional output if you switch the browser to debug mode.